Persona.Merger.Tests/Parser/P5RTblPatcherTests.cs

```diff
@@ -37,4 +38,40 @@ public void PatchTbl_Item_Extend()
         var patched = patcher.Apply(new List<TblPatch>() { patch });
         Assert.Equal(after, patched);
     }
+    
+    [Fact]
+    public void PatchTbl_Generic()
+    {
+        var patches = new List<TblPatch>();
+
+        var original = File.ReadAllBytes(P5RAssets.PDDBefore);
+        var after = File.ReadAllBytes(P5RAssets.PDDAfter);
+        var after2 = File.ReadAllBytes(P5RAssets.PDDAfter2);
+        var after3 = File.ReadAllBytes(P5RAssets.PDDAfter3);
+        var patcher = new GenericPatcher(original);
+        patches.Add(patcher.GeneratePatchGeneric(after, 4));
+        patches.Add(patcher.GeneratePatchGeneric(after2, 4));
+        var patched = patcher.ApplyGeneric(patches);
+
+        Assert.Equal(patched, after3);
+    }
+    
+    [Fact]
+    public void PatchTbl_VISUAL()
+    {
+        var patches = new List<TblPatch>();
+
+        var original = File.ReadAllBytes(P5RAssets.VisualBefore);
+        var after = File.ReadAllBytes(P5RAssets.VisualAfter);
+        var after2 = File.ReadAllBytes(P5RAssets.VisualAfter2);
+        var after3 = File.ReadAllBytes(P5RAssets.VisualAfter3);
+        var patcher = new P5RTblPatcher(original, TblType.Visual);
+        patches.Add(patcher.GeneratePatch(after));
+        patches.Add(patcher.GeneratePatch(after2));
+        var patched = patcher.Apply(patches);
+
+        File.WriteAllBytes("D:\\Game Modding\\R2\\Loader\\VISUAL.TBL", patched);
```

Comment:
This is writing to a location on your disk; and would fail for others because directory does not exist.

Reply:
Removed this line as this should not have been here, accidental leftover when adding the unit test.